website/docs/docs/contributing/building-a-new-adapter.md

```diff
@@ -153,15 +153,15 @@ For example:
 
 </File>
 
-#### get_status(cls, cursor)
+#### get_response(cls, cursor)
 
-get_status is a classmethod that gets a cursor object and returns the status of the last executed command. The return value is one of the values returned from `execute()` and is logged. You can just return 'OK' or similar if your database does not provide status.
+get_response is a classmethod that gets a cursor object and returns the status of the last executed command. The return value is one of the values returned from `execute()` and is logged. You can just return 'OK' or similar if your database does not provide status.
```

Comment:
I realized there is a subtle distinction to make here:
```suggestion
get_response is a classmethod that gets a cursor object and returns adapter-specific information about the last executed command. Ideally, the return value is an `AdapterResponse` object that includes items such as `code`, `rows_affected`, `bytes_processed`, and a summary `_message` for logging to stdout. Or, get_response can just return a string `'OK'` if your connection cursor does not provide richer metadata.
```
The basic implementation below still works, so just a simple rename there is fine